src/main/kotlin/com/vk/admstorm/actions/CreateHasteAction.kt

```diff
@@ -20,10 +20,12 @@ class CreateHasteAction : AdmActionBase() {
 
         executeOnPooledThread {
             val link = HastebinService.getInstance(e.project!!).createHaste(copyText)
-            MyUtils.copyToClipboard(link)
-            AdmNotification()
-                .withTitle("Link to hastebin copied to clipboard")
-                .show()
+            if (link != null) {
```

Comment:
Well, no... If something goes wrong, you have to report it

Reply:
This applies to all changes

Reply:
Done